odd-platform-ui/src/components/DataEntityDetails/Metadata/MetadataCreateForm/MetadataCreateFormItem/MetadataCreateFormItem.tsx

```diff
@@ -156,6 +156,7 @@ const MetadataCreateFormItem: React.FC = () => {
             renderInput={params => (
               <AppInput
                 {...params}
+                dataQAId='Add_custom_metadata_input'
```

Comment:
should `A` in "'Add_custom_metadata_input'" be in lowercase?

Reply:
Corrected